content/css/jenkins.css

```diff
@@ -1380,6 +1380,19 @@ p.details {
   text-align: center;
 }
 
+.imageblock.boxshadow img {
+  margin: 5px 5px 5px 5px;
+  -moz-box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
+  -webkit-box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
+  box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
+}
```

Comment:
This CSS addition supports block image references. Read more about this in [this comment on WEBSITE-416](https://issues.jenkins-ci.org/browse/WEBSITE-416?focusedCommentId=317287&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-317287).